libavif.pc.cmake

```diff
@@ -1,10 +1,10 @@
 prefix=@CMAKE_INSTALL_PREFIX@
-exec_prefix=${prefix}/bin
-libdir=${prefix}/@CMAKE_INSTALL_LIBDIR@
-includedir=${prefix}/include
+exec_prefix=${prefix}
+libdir=${exec_prefix}/@CMAKE_INSTALL_LIBDIR@
+includedir=${prefix}/@CMAKE_INSTALL_INCLUDEDIR@
 
 Name: @PROJECT_NAME@
 Description: Library for encoding and decoding .avif files
 Version: @PROJECT_VERSION@
 Libs: -L${libdir} -lavif
-Cflags: -I${includedir}@AVIF_PKG_CONFIG_EXTRA_CFLAGS@
+Cflags: -I${includedir}/avif@AVIF_PKG_CONFIG_EXTRA_CFLAGS@
```

Comment:
Just to make sure, is it recommended for users of libavif to use `#include <avif/avif.h>` or `#include <avif.h>`?

I see in https://github.com/AOMediaCodec/libavif/blob/master/apps/avifenc.c#L4 that @joedrago uses the former, so in that case, this change would be unnecessary or incorrect

Reply:
Good point. The apps and tests in libavif use `#include "avif/avif.h"`.

Reply:
Obviously extraneous then, I'll update the PR.

Reply:
`avif/avif.h` is the correct and intended include.